packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -294,16 +302,14 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   }
 
   &.mdc-textfield--dense {
-    height: 44px;
```

Comment:
Removing this seems to add roughly 16px to the bottom of the textfield box. Is this intentional?

Reply:
Yep. The spec doesn't say to change the height of dense text fields. I don't know why this was added in the first place.